src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/scarlet_monastery/boss_mograine_and_whitemane.cpp

```diff
@@ -422,6 +427,12 @@ struct boss_high_inquisitor_whitemaneAI : public ScriptedAI
         else
             m_uiHolySmite_Timer -= uiDiff;
 
+        if(urand(1, 1000) == 1){
```

Comment:
No idea what the actual values are, unfortunately, so I'm just giving it a 1:1000 chance of occurring. According to the comments it happens very rarely, and I think that should be given like this, but it's probably better to replace this with a known value, once it exists.